website/app/main.js

```diff
@@ -18,7 +18,7 @@ window.MacMessenger = {
   showSettings: function() {
     this.openGearMenu();
     document.querySelector(
-      'div.uiLayer.uiContextualLayerPositioner#js_1 > div.uiContextualLayer a'
+      'div.uiLayer.uiContextualLayerPositioner#js_2 > div.uiContextualLayer a'
```

Comment:
@CvX Hmm, they are definitely all auto-generated. It looks like the class `.uiContextualLayerPositioner` has been a consistent hook then. We can do the reverse order. Select this and use the id to find the thing targeting it. I've got no bright ideas short of just asking them to leave class name hooks for external modification.